pkg/cmd/target/drop/options.go

```diff
@@ -0,0 +1,82 @@
+/*
+SPDX-FileCopyrightText: 2021 SAP SE or an SAP affiliate company and Gardener contributors
+
+SPDX-License-Identifier: Apache-2.0
+*/
+
+package drop
+
+import (
+	"fmt"
+	"strings"
+
+	"github.com/gardener/gardenctl-v2/internal/util"
+	"github.com/gardener/gardenctl-v2/pkg/cmd/base"
+	"github.com/gardener/gardenctl-v2/pkg/target"
+
+	"github.com/spf13/cobra"
+)
+
+// TargetKind is representing the type of things that can be targeted
+// by this cobra command. While this may sound stuttery, the alternative
+// of just calling it "Kind" is even worse, hence the nolint.
+// nolint
+type TargetKind string
+
+const (
+	TargetKindGarden  TargetKind = "garden"
+	TargetKindProject TargetKind = "project"
+	TargetKindSeed    TargetKind = "seed"
+	TargetKindShoot   TargetKind = "shoot"
+)
+
+var (
+	AllTargetKinds = []TargetKind{TargetKindGarden, TargetKindProject, TargetKindSeed, TargetKindShoot}
+)
```

Comment:
remove the duplicate constants and `TargetKind` type and do something like:
```go
	targetcmd "github.com/gardener/gardenctl-v2/pkg/cmd/target"
	"github.com/gardener/gardenctl-v2/pkg/target"

	"github.com/spf13/cobra"
)

var (
	AllTargetKinds = []targetcmd.TargetKind{targetcmd.TargetKindGarden, targetcmd.TargetKindProject, targetcmd.TargetKindSeed, targetcmd.TargetKindShoot}
)
```
this should work, or not?

Reply:
I will double check this. I tried something similar and got some cycle depedency issue I will try again and get in touch with you if I face similar issues.

Reply:
refactored as discussed